src/components/PeopleLink/PeopleLink.tsx

```diff
@@ -0,0 +1,50 @@
+import classNames from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+import { Person } from '../../types';
+
+type Props = {
+  person: Person;
+};
+
+export const PeopleLink: React.FC<Props> = ({ person }) => {
+  const { name, sex, born, died, motherName, fatherName, mother, father } =
+    person;
+  const { slug } = useParams();
+  const noParent = '-';
+
+  return (
+    <tr
+      data-cy="person"
+      className={classNames({ 'has-background-warning': slug === person.slug })}
+    >
+      <td>
+        <Link
+          to={`/people/${person.slug}`}
+          className={classNames(sex === 'f' ? 'has-text-danger' : '')}
```

Comment:
Use the classnames library rules for add classes with condition, fix it everywhere